js/Application.js

```diff
@@ -242,7 +242,14 @@ define(
 					}
 				});
 			}
-
+      checkOAuthError() {
+        let hash = window.location.hash;
+        if (hash) {
+          if (hash.includes("oauth_error_email")) {
```

Comment:
`hash && hash.includes("oauth_error_email")` instead of nested `if`

Reply:
fixed